emulator/src/test/java/com/scalar/client/tool/emulator/EmulatorTerminalTest.java

```diff
@@ -91,4 +91,109 @@ public void input_withSpacesForFlagInput_ShouldReturnProperParamsArrayProperly()
     // Assert
     Assert.assertArrayEquals(paramsArray, mockedParamsArray);
   }
+
+  @Test
+  public void get_withSpacesForInput_ShouldReturnProperParamsArrayProperly() {
```

Comment:
```suggestion
  public void get_WithSpacesForInput_ShouldReturnProperParamsArrayProperly() {
```
The same for the others.

Reply:
Thank you for detailed review.
The test case method name updated.